README.md

```diff
@@ -36,7 +36,7 @@ developer communities.
 * Self-play mechanism for training agents in adversarial scenarios
 * Train memory-enhanced agents using deep reinforcement learning
 * Easily definable Curriculum Learning and Generalization scenarios
-* Built-in support for Imitation Learning
+* Built-in support for [Imitation Learning](https://github.com/Unity-Technologies/ml-agents/blob/master/docs/Training-Imitation-Learning.md) through Behavioral Cloning or Generative Adversarial Imitation Learning
```

Comment:
Not sure it is recommended to use the full URL. Maybe just do 
```suggestion
* Built-in support for [Imitation Learning](docs/Training-Imitation-Learning.md) through Behavioral Cloning or Generative Adversarial Imitation Learning
```

Reply:
For the README it has to link to the latest_release files (and use the full path)... but you're right it was wrong. Updated.